vortex-array/src/builders/varbinview_builder.rs

```diff
@@ -1,171 +0,0 @@
-use std::any::Any;
```

Comment:
for other readers - this file is just a zombie, it seems to be a copy of `vortex-array/src/builders/varbinview.rs`.